include/z64camera.h

```diff
@@ -1010,7 +1010,7 @@ typedef struct {
 typedef struct {
     /* 0x00 */ Vec3f initalPos;
     /* 0x0C */ s16 animTimer;
-    /* 0x10 */ InfiniteLine sceneCamPosPlayerLine;
+    /* 0x10 */ InfiniteLine bgCamPosPlayerLine;
```

Comment:
```suggestion
    /* 0x10 */ InfiniteLine eyeAndDirection;
```
or leave it as is, not a big deal for now